<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExecPrograms</class>
 <widget class="QWidget" name="ExecPrograms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Run the program before the call:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="programStart1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>#1</string>
   </property>
  </widget>
  <widget class="QToolButton" name="chooseProgramStart1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Program selection</string>
   </property>
   <property name="whatsThis">
    <string>Program selection</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="chooseProgramStart2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Program selection</string>
   </property>
   <property name="whatsThis">
    <string>Program selection</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>#2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="programStart2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Run the program after the call:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="chooseProgramEnd1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Program selection</string>
   </property>
   <property name="whatsThis">
    <string>Program selection</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>#2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>#1</string>
   </property>
  </widget>
  <widget class="QToolButton" name="chooseProgramEnd2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Program selection</string>
   </property>
   <property name="whatsThis">
    <string>Program selection</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="programEnd1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="programEnd2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
